continuous_integration/install.sh

```diff
@@ -35,7 +35,7 @@ print_conda_requirements() {
     #   - for scikit-learn, SCIKIT_LEARN_VERSION is used
     TO_INSTALL_ALWAYS="pip nose pytest"
     REQUIREMENTS="$TO_INSTALL_ALWAYS"
-    TO_INSTALL_MAYBE="python numpy scipy matplotlib scikit-learn pandas flake8 networkx configobj"
+    TO_INSTALL_MAYBE="numpy scipy matplotlib scikit-learn nilearn nipype pandas flake8 lxml joblib networkx configobj"
```

Comment:
lxml ?

Reply:
Again copied from nilearn. I figured I would remove the extra packages once the required changes are made to pypreprocess and Travis no longer fails.